LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 The Envoyproxy Authors
```

Comment:
I am curious if the year should be updated every year?

Reply:
It should get updated each time the file is touched.

Reply:
Read the paragraph above carefully, this part should be never changed: https://github.com/envoyproxy/envoy/pull/737#discussion_r112274371